docs/src/main/asciidoc/tika-guide.adoc

```diff
@@ -0,0 +1,88 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using Apache Tika
+
+include::./attributes.adoc[]
+:extension-name: Apache Tika
```

Comment:
I don't see the purpose of this variable. It makes things less readable and there's no chance we will change it.

And in fact, I don't think you use it.

Reply:
Ok, will remove it